### DISCUSSION TITLE
How to change the interface language?

### DISCUSSION THREAD

**Discussion by OleksandrMak** [inventree/InvenTree#4413] [Q&A]
How can I change language of web interface of Inventree? For example for 3 different users - 3 different languages. I didn't found language set in settings.

Also if there is no my translation how can I make translation for Inventree?

**Comment by matmair** [comment]
hi there @OleksandrMak 
Go to Settings->User Settings->Display Settings->Language to change the language (for you).
Translations can be submitted [here](https://crowdin.com/project/inventree) .

**Reply by SchrodingersGat** [reply]
https://docs.inventree.org/en/latest/settings/user/#display-settings
